packages/ui/package.json

```diff
@@ -98,6 +98,7 @@
   "devDependencies": {
     "@calcom/config": "*",
     "@calcom/tsconfig": "*",
+    "@testing-library/user-event": "^14.6.1",
```

Comment:
Where are you using this?

Reply:
you'll also have to update yarn.lock file if we need this

Reply:
> Where are you using this?

This is used in some tests that are located outside of UI. Makes sense for this to be a dev depencanies so other places in the app can write tests for UI components without needing this dependancy installed in their package

Reply:
Its also used in a few places

packages/ui/components/editor/Editor.test.tsx
packages/ui/components/editor/plugins/ToolbarPlugin.test.tsx
color picker , attribute form test